src/Education.UnitTests/SimpleRuleHelpXamlBuilderTests.cs

```diff
@@ -148,7 +151,7 @@ private static bool ProcessResource(string fullResourceName)
                     // Note: this is a quick way of getting the size of the document. Serializing the doc to a string
                     // and checking the length takes much longer (around 25 seconds)
                     var docLength = doc.ContentStart.DocumentStart.GetOffsetToPosition(doc.ContentEnd.DocumentEnd);
-                    Console.WriteLine($"{jsonRuleInfo.FullRuleKey}: size = {docLength}");
+                    // Console.WriteLine($"{jsonRuleInfo.FullRuleKey}: size = {docLength}");
```

Comment:
Does this need to be commented out?

Reply:
Forgot to revert it before the commit. TBH, this line is pretty useless and was preventing me from seeing the actual errors so at one point I was considering removing it. What do you think?

Reply:
If it's not helping, remove it.